data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition: Alpha Encounter"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		has "remembrance day: offered"
+		random < 1
+	source
+		government "Coalition"
+		not attributes "station"
+		not planet "Secret Sky" "Remote Blue" "Bloptab's Furnace" "Ahr"
+	on offer
+		log "Minor People" "Alitis" `Alitis, formerly known as Athis, is an Alpha who lives in the Coalition. Centuries ago, he stole a jump drive from an alien ship and used it to explore, but after some sort of encounter in the galactic east, he decided to give up on his former life.`
+		conversation
+			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground. You try to get up, but whoever attacked you pins you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`
+			`	Your eyes don't meet for long, though, as he looks to his side to see an Arach that swoops in with a tackle. He lets go of your arms in an attempt to block the attack, but is too late. As you get up, they wrestle for a while, but it doesn't take long for the Arach Heliarch to restrain the Alpha. All eight legs firmly hold his arms to his torso as the Arach rests her entire body weight on the Alpha's back. A Saryd Heliarch approaches you, telling you to keep your distance. He then looks at the Alpha and asks, "What reason for attacking this person, did you have?"`
+			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`
```

Comment:
For that first paragraph I'd drop the bit about the quicksteps, I'd expect an Alpha to be more capable at catching someone by surprise, and quick footsteps in a spaceport marketplace shouldn't be a source of concern in the Coalition, where one expects to be safe, and wouldn't assume any quick footsteps would mean a potential attacker.

Most of my problems are with the suggested second paragraph.

- I always forget this so correct me if I've got this wrong again, but aren't all Alphas male? If so then `The Alpha is a man` is redundant.
- I feel it's more a testament to the Arach's abilities if we don't explicity say "even though Alphas are so strong the Arach can restrain this one pretty easily."
- I don't see the need to change the hold the Arach has on the Alpha, and this one seems like a worse version of the "six legs on him, two on the ground" that you proposed in the other comment. How would she restrain each of his legs with one of her own, wouldn't her abdomen get in the way or make that an awkward hold?

The `snarling expression` part doesn't seem necessary to me.

That entire last paragraph, no. The necklace can be mentioned earlier on, the rest of the paragraph feels almost like a fourth wall break, "oh wow look at this, this Alpha doesn't look like a big meanie psycho, weird huh?"

Reply:
Re 1) So, the Alpha still catches you by surprise, it's too late to do anything about it by the time you hear the last two steps approaching you...

Re 2)
- Alphas can be female, it is simply that female alphas cannot produce viable alpha offspring.
- I know we spend a non-trivial amount of time on the discord talking about alphas and their abilities, but it is worth noting that actually in-game the player is not frequently encountering references to them. Reminding the player occasionally that they're meant to be scary af doesn't hurt, especially when you're then explicitly overpowering them.
- The first hold you had is literally impossible to maintain, so it needs to change. Precisely how you imagine this hold to go down will vary by your mental image of how fat or big the Arach is, but it is worth noting that most spiders can lift their abdomen up to get it clear of the ground when they're doing fancy footwork.

Do you want your alpha to have emotion or not? Pick a side. At present you have dispassionate coldness here and then emotion later on. 

The necklace absolutely cannot be mentioned earlier on. Why in the name of all that is holy are you noticing a necklace while tackled to the ground and struggling for your life? You've got 2-dozen things to think about before you get to that point, and including it where you have detracts from that flow. Your first opportunity to get a good look at him is after he's been pinned and you've stood up. Where else would you include any description that points to him not being entirely what you expect?

Reply:
Again, why would I be alarmed by some footsteps when I'm in a very safe location?

`I know we spend a non-trivial amount of time on the discord talking about alphas and their abilities, but it is worth noting that actually in-game the player is not frequently encountering references to them. Reminding the player occasionally that they're meant to be scary af doesn't hurt, especially when you're then explicitly overpowering them.`
I dunno, seems kind of out of place. If we're doing it I'd like if it wasn't said so explicitly to the player.

`The first hold you had is literally impossible to maintain`
I really doubt that.

`Do you want your alpha to have emotion or not?`
Having emotion doesn't mean just one state of being, he's not going to be a robot but he's not going to be bawling his eyes out either, that's exactly why the start of the conversation will be changed, as in that he's exceedingly casual.

Necklace can be mentioned once the Arach restrains the Alpha, or when he perks up his head to ask you the question.

Reply:
`why would I be alarmed` It doesn't say that you're alarmed...

`I'd like if it wasn't said so explicitly to the player` that's probably fair, it can possibly be rephrased

`I really doubt that.` I can draw you a diagram if you'd like.

`as in that he's exceedingly casual` then it's simple enough to take off the snarling face and just swap out for an intense look

Reply:
What would you be reacting to if you weren't alarmed? Does first last just turn to meet whoever is passing by in a rush?

I don't need a diagram to know the Alpha isn't getting out of the hold of a giant spider holding his arms to his torso while her entire body weight rests on him.

I'd just go without a remark, having him just looking at you, but that would be the better of the two if I were to pick a remark to go with.

Reply:
`Does first last just turn to meet whoever is passing by in a rush?` Probably not, but it's a natural human response to at least glance at someone if they suddenly step very near to you. That in itself is a reaction, if only a small one.

Adjusted suggestion from this convo:

```suggestion
			`You're window-browsing a shop in the spaceport when you catch the scuff of two quick footsteps coming towards you. Before you can glance in that direction though you're suddenly tackled into the ground by someone astonishingly human-shaped. You try to get up, but whoever attacked you pins you down, pressing your face into the ground and holding your arms behind your back to restrain you. As you struggle against your attacker you manage to catch sight of long hair and then, with a flash of concern, a glimpse of bright green skin - an Alpha.`
			`	You are not pressed for long though, you feel their attention shift to the side you cannot see, as an Arach swoops in with a tackle. They lets go of your arms in an attempt to block the attack, but it does not prevent the Arach from bowling them clear off of you where the two begin to wrestle in your line of sight. The Alpha is a man and, surprisingly to you, despite his strength it doesn't take long for the Arach Heliarch to restrain your attacker. The Arach firmly pins his arms to his torso with four legs, traps his legs with two, and uses her last pair to lever herself until she rests her entire body weight into pinning the Alpha.`
			`	A Saryd Heliarch approaches and tells you to keep your distance before interposing himself partially between you and your assailant. He then looks at the Alpha and asks through his translator box, "What reason for attacking this person, did you have?"`
			`	The Alpha is still struggling to try and get out of the Arach's grip, and you notice an intricate gold necklace which has splayed across his shoulder and onto the floor in the scuffle, jangling in the quiet every time he struggles as now everyone else has cleared out of the area and is giving the scene a wide berth. As you get to your feet he ignores the Saryd's question and looks at you intensely. "You're with the Navy, aren't you? Finally figured out I was here, thought you'd finish the job?"`
```